js/pro/kucoin.js

```diff
@@ -647,7 +647,7 @@ module.exports = class kucoin extends kucoinRest {
         //         'side': 'buy',
         //         'orderId': '6249167327218b000135e749',
         //         'type': 'canceled',
-        //         'orderTime': 1648957043065280224,
```

Comment:
Didn't see they update the `orderTime` https://docs.kucoin.com/#private-order-change-events lol

Reply:
erm this thing is weird. i not sure why. because if i am creating an order, i get a truncated timestamp as a result as it is returning in milliseconds then multiplied by 0.000001 
but for cancelling order it would return a normal nano seconds as usual.